garage/envs/mujoco/gather/gather_env.py

```diff
@@ -414,10 +418,14 @@ def stop_viewer(self):
 
     def render(self, mode='human', close=False):  # pylint: disable=R1710
         if mode == 'rgb_array':
-            self.get_viewer().render()
-            data, width, height = self.get_viewer().get_image()
-            return np.fromstring(
-                data, dtype='uint8').reshape(height, width, 3)[::-1, :, :]
+            viewer = self.get_viewer()
+            self.wrapped_env.render()
+            img = viewer.read_pixels(width=self.render_width,
+                                     height=self.render_height,
+                                     depth=False)
+            img = img[::-1, :, :]
```

Comment:
Maybe adding a comment specifying the original image was upside-down and this line flips it can be helpful when looking back on this code